tests/installation/first_boot.pm

```diff
@@ -23,7 +23,7 @@ sub handle_login {
     if (get_var('DM_NEEDS_USERNAME')) {
         type_string "$username\n";
     }
-    if (check_var('DESKTOP', 'gnome')) {
+    if (check_var('DESKTOP', 'gnome') || (check_var('DESKTOP', 'lxde') && check_var('VERSION', '42.1'))) {
```

Comment:
what do you mean with version comparison ?
this is here only because this specific DM on lxde 42.1 does need to select user, such case is not in version before or after (It is maybe silly, but it is so)